src/components/CommentDeleter.jsx

```diff
@@ -0,0 +1,42 @@
+import { useState } from "react";
+import { deleteComment } from "../api";
+import ArticleComments from "./ArticleComments";
+
+export default function CommentDeleter({
```

Comment:
`DeleteCommentButton` may be a better name for this, it would add a bit more semantic meaning